server/planning/__init__.py

```diff
@@ -16,6 +16,10 @@
 from .coverage import CoverageResource, CoverageService
 from .locations import LocationsResource, LocationsService
 
+# import feed parsers and services
+from .feed_parsers.ics_2_0 import IcsTwoFeedParser  # NOQA
+from .feeding_services.event_file_service import EventFileFeedingService  # NOQA
```

Comment:
I know this is an ordinary thing in SD, but I would avoid to use `# NOQA` there, and would register them directly here, in `init_app`. Just saying, I will do it myself if you agree